package-lock.json

```diff
@@ -2687,7 +2687,7 @@
     "crypto-browserify": {
       "version": "3.12.0",
       "resolved": "https://registry.npmjs.org/crypto-browserify/-/crypto-browserify-3.12.0.tgz",
-      "integrity": "sha512-fz4spIh+znjO2VjL+IdhEpRJ3YN6sMzITSBijk6FK2UvTqruSQW+/cCZTSNsMiZNvUeq0CqurF+dAbyiGOY6Wg==",
+      "integrity": "sha1-OWz58xN/A+S45TLFj2mCVOAPgOw=",
```

Comment:
@AlasdairSwan Hey, do know you why this change would happen? I last noticed this change on edx-mktg. I cannot remember if we decided to take in this change on marketing site or not? Can you help us ring a bell here?